app/models/chargeback_container_project.rb

```diff
@@ -62,19 +64,38 @@ def self.build_results_for_report_ChargebackContainerProject(options)
       @data_index.store_path(:container_project, :by_group_id, g.id, g.container_project || g.old_container_project)
     end
 
+    if @options[:groupby_tag]
+      @tags_for_category = Classification.find_by_name(@options[:groupby_tag]).entries
+      @tags_for_category.each do |tag|
+        ContainerProject.with_any_tags([tag.tag_id]).each do |project|
+          @data_index.store_path(:tag, :by_project_id, project.id, tag)
+        end
+      end
+    end
```

Comment:
I think you want to use `find_tagged_with(:all => ...)` [[ref]](https://github.com/ManageIQ/manageiq/blob/0183d8ee1a44a77ca8c26e4762aaea3d33b89315/lib/extensions/ar_taggable.rb#L35) instead of `with_any_tag`. wondering if `with_any_tag` should become private

would have been nice if you could just pass category into there and not passed the actual tags